doc/index.rst

```diff
@@ -2,14 +2,11 @@
 
 .. title:: Matplotlib: Python plotting
 
-Matplotlib is a Python 2D plotting library which produces publication quality
-figures in a variety of hardcopy formats and interactive environments across
-platforms.  Matplotlib can be used in Python scripts, the Python and IPython_
-shells, the Jupyter_ notebook, web application servers, and four graphical user
-interface toolkits.
+Matplotlib: 2D visualization in Python
+--------------------------------------
 
-.. _IPython: http://ipython.org
-.. _Jupyter: http://jupyter.org
+Matplotlib supports your visualization tasks, from simple one-off plotting to
```

Comment:
#
```suggestion
Matplotlib supports visualization tasks, from simple plots to
```

Reply:
Took "plot", kept "your"; I explicitly would like to address and include the user in this very first. It's not only what we do, but it addresses the user's needs.